src/tile-board/Tileboard.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Tile from './Tile';
+
+function TileBoard() {
+  const arrayGrid = createArrayGrid();
+  
+  return (
+    <div className='container text-center mt-5'>
+      {
+        // Maps over the array grid, adding a <div> element that wraps around each row.
+        arrayGrid.map((row, key) => {
+          return <div key={key}>{row}</div>;
+        })
+      }
+    </div>
+  )
+}
+
+/* 
+  Uses two nested for-loops that loops through each tile space 
```

Comment:
When writing comments like this, I try to think about a few things:

1. Who is this comment for?
2. What do they need to know (that they can't otherwise determine by looking at the code itself)?
3. Do I need a comment here at all?
 
Code comments have an associated cost:
- they are part of the source code that someone else working on the project will need to read and understand
- they need to be maintained - if someone changes the code but doesn't update the comment, it could become outdated and inaccurate which might cause someone else confusion in the future

In this case, a description of the "how" (for loops), and the "what" (nested array) are both something that I would ordinarily not include in a comment like this if I'm assuming the audience is reasonably proficient with JavaScript.

Occasionally, if there's a particularly tricky construction or complex data structures, I might determine that it's useful to summarize some of the "how" and "what", but I find most code is readable enough on its own.

If you are going to assume that the grid is always a square, you could change this function to take an `size` number argument (that perhaps defaults to 5) and use that in your nested loops.